src/classes/USER_UserService_TEST.cls

```diff
@@ -0,0 +1,240 @@
+		List<User> newUsers = createUsers();
+    	Set<Id> sysAdminIds = new Map<Id, User>(newUsers).keySet();
+
+    	// Create and insert an Error_Settings__c record with an IsFrozen User to update via setErrorNotificationToAllSysAdmins accordingly
+    	UTIL_CustomSettingsFacade.getErrorSettingsForTests(new Error_Settings__c(Error_Notifications_On__c = true, 
+            Error_Notifications_To__c = newUsers[1].Id, Store_Errors_On__c = true));
+    	Error_Settings__c errorSettings = UTIL_CustomSettingsFacade.getErrorSettings();
+		Database.insert(errorSettings, true);
+
+    	Test.startTest();
+        USER_UserService.applySysAdminIsActiveIsFrozenStatusChanges(sysAdminIds);
+        Test.stopTest();
+
+        Error_Settings__c errorSettingsSysAdmin = UTIL_CustomSettingsFacade.getErrorSettings();
+        System.assertEquals(ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS, errorSettingsSysAdmin.Error_Notifications_To__c);        
+	}
+
+
+	/*********************************************************************************************************
+    * @description Tests setting the NPSP Settings Error Notification Recipients to All Sys Admins.
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testSetErrorNotificationToAllSysAdmins() {
+    	List<User> newUsers = createUsers();
+    	Set<Id> sysAdminIds = new Map<Id, User>(newUsers).keySet();
+
+    	// Create and insert an Error_Settings__c record with an IsFrozen User to update via setErrorNotificationToAllSysAdmins accordingly
+    	UTIL_CustomSettingsFacade.getErrorSettingsForTests(new Error_Settings__c(Error_Notifications_On__c = true, 
+            Error_Notifications_To__c = newUsers[1].Id, Store_Errors_On__c = true));
+    	Error_Settings__c errorSettings = UTIL_CustomSettingsFacade.getErrorSettings();
+		Database.insert(errorSettings, true);
+		
+		Test.startTest();
+        USER_UserService.setErrorNotificationToAllSysAdmins(sysAdminIds);
+        Test.stopTest();
+
+        Error_Settings__c errorSettingsSysAdmin = UTIL_CustomSettingsFacade.getErrorSettings();
+        System.assertEquals(ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS, errorSettingsSysAdmin.Error_Notifications_To__c);
+    }
+
+
+   	/*********************************************************************************************************
+    * @description Tests if User is the owner of Scheduled Jobs and calls the methods that logs an error and sends an email.
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testHandleScheduledJobsSubmittedByInActiveIsFrozenSysAdmin() {
+    	List<User> newUsers = createUsers();
+    	Set<Id> sysAdminIds = new Map<Id, User>(newUsers).keySet();
+
+    	Test.startTest();
+    	UTIL_MasterSchedulableHelper.setScheduledJobs();
```

Comment:
won't this logic create the scheduled jobs using the current running user, rather than one of the users you have created.  thus running your service shouldn't affect the jobs.  seems like you need two tests.  one that verifies that deactivating a user who isn't the owner of the jobs has no affect on the jobs (userId better be what it originally was), and another test where you schedule the jobs as a user who you the freeze, and then you verify the jobs gets cleared?  resaved with another user?

Reply:
Updated the test to create the jobs as the User being modified. Thanks!